test/testdrive/mz-support-privileges.td

```diff
@@ -0,0 +1,37 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+# Tests that assert the privileges that are assumed to be always granted to #
+# the mz_support user. This test can be rewritten to validate the output of the
+# `connection=mz_support` command once we have `SET ROLE` working.
+
+$ postgres-connect name=mz_support url=postgres://mz_support:materialize@${testdrive.materialize-internal-sql-addr}
+
+> CREATE SOURCE auction_house FROM LOAD GENERATOR AUCTION FOR ALL TABLES WITH (size = '1');
```

Comment:
@MaterializeInc/testing the fact that I use `WITH (size = '1')` here might break CI (although strangely it didn't break the CI tests for this PR). Feel free to remove this suffix.